contrib/src/test/java/gov/nasa/jpl/aerie/contrib/models/SampledResourceTest.java

```diff
@@ -0,0 +1,31 @@
+package gov.nasa.jpl.aerie.contrib.models;
+
+import gov.nasa.jpl.aerie.merlin.framework.junit.MerlinExtension;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import static gov.nasa.jpl.aerie.merlin.framework.ModelActions.delay;
+import static gov.nasa.jpl.aerie.merlin.protocol.types.Duration.SECOND;
+import static gov.nasa.jpl.aerie.merlin.protocol.types.Duration.SECONDS;
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(MerlinExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class SampledResourceTest {
+  final Accumulator accumulator = new Accumulator(0.0, 1.0);
+  final SampledResource<Double> sampledResource = new SampledResource<>(() -> accumulator.get(), $ -> $, 10.0);
```

Comment:
What does `$->$` mean?

Adjacently, based on the test, this line appears to be making a `SampledResource` that accumulates at a rate of once per second but whose value is only updated every ten seconds. Is that correct?

Reply:
`$ -> $` is the identity function - the `SampledResource` (more specifically the underlying register), asks for a way to duplicate the contained value. If the type contained in the SampledResource were mutable, this function would be expected to make a new copy. Since it's a `Double`, the identity function suffices here